test/parallel/test-fs-util-validateoffsetlengthwrite.js

```diff
@@ -22,22 +22,6 @@ const { kMaxLength } = require('buffer');
   );
 }
 
-// RangeError when byteLength > kMaxLength, and length > kMaxLength - offset .
-{
-  const offset = kMaxLength;
-  const length = 100;
-  const byteLength = kMaxLength + 1;
-  common.expectsError(
-    () => validateOffsetLengthWrite(offset, length, byteLength),
-    {
-      code: 'ERR_OUT_OF_RANGE',
-      type: RangeError,
-      message: 'The value of "length" is out of range. ' +
-               `It must be <= ${kMaxLength - offset}. Received ${length}`
-    }
-  );
-}
```

Comment:
Does this test have to be removed? To me it looks like it should still pass but maybe with a slightly different message?

Reply:
I think it's redundant with the test below it, right? This one tested the case of `byteLength > kMaxLength`, which is no longer a requirement.